docs/yaml/objects/cfg_data.yaml

```
@@ -41,8 +41,16 @@ methods:
       type: str
       description: The name of the variable to set
     value:
-      type: bool
-      description: The value to set as either `1` or `0`
+      type: bool | int
+      description: |
+        The value to set as either `1` or `0`
+
+        Since 0.62.0 passing a number is deprecated, use only booleans or the
+        `.set` method instead.
```

I would still like this to not claim that it is only deprecated since 0.62

```suggestion
        Passing a number was never intended to work, and is deprecated and will be
        removed in a future version of meson. Use only booleans or the
        `.set` method instead.
```

---

reworked again. I think having 0.62 in there is important so that users know why they are suddenly getting this warning